<ui version="4.0" >
 <class>UserPreferences</class>
 <widget class="QDialog" name="UserPreferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >:/icons/icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="appearance" >
      <attribute name="title" >
       <string>Look &amp;&amp; Feel</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Font Settings</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&amp;Font:</string>
              </property>
              <property name="buddy" >
               <cstring>m_fontComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QFontComboBox" name="m_fontComboBox" >
              <property name="currentFont" >
               <font>
                <family>Courier New</family>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>&amp;Size</string>
              </property>
              <property name="buddy" >
               <cstring>m_fontSize</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QSpinBox" name="m_fontSize" >
              <property name="minimum" >
               <number>4</number>
              </property>
              <property name="maximum" >
               <number>22</number>
              </property>
              <property name="value" >
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QGroupBox" name="mp_foregroundGrpBox" >
           <property name="title" >
            <string>Foreground colors</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Text:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="ColorButton" name="mp_color0_39Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>Black:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="ColorButton" name="mp_color30Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Red:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="ColorButton" name="mp_color31Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Green:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="ColorButton" name="mp_color32Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Yellow:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="ColorButton" name="mp_color33Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>Blue:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" >
             <widget class="ColorButton" name="mp_color34Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="0" >
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>Magenta:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" >
             <widget class="ColorButton" name="mp_color35Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="7" column="0" >
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>Cyan:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1" >
             <widget class="ColorButton" name="mp_color36Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="8" column="0" >
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>White:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1" >
             <widget class="ColorButton" name="mp_color37Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="mp_backgroundGrpBox" >
           <property name="title" >
            <string>Background colors</string>
           </property>
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Default background:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="ColorButton" name="mp_color0_49Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_19" >
              <property name="text" >
               <string>Black:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="ColorButton" name="mp_color40Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Red:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="ColorButton" name="mp_color41Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_17" >
              <property name="text" >
               <string>Green:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="ColorButton" name="mp_color42Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QLabel" name="label_20" >
              <property name="text" >
               <string>Yellow:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="ColorButton" name="mp_color43Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QLabel" name="label_15" >
              <property name="text" >
               <string>Blue:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" >
             <widget class="ColorButton" name="mp_color44Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="0" >
             <widget class="QLabel" name="label_18" >
              <property name="text" >
               <string>Magenta:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" >
             <widget class="ColorButton" name="mp_color45Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="7" column="0" >
             <widget class="QLabel" name="label_16" >
              <property name="text" >
               <string>Cyan:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1" >
             <widget class="ColorButton" name="mp_color46Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item row="8" column="0" >
             <widget class="QLabel" name="label_14" >
              <property name="text" >
               <string>White:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1" >
             <widget class="ColorButton" name="mp_color47Btn" >
              <property name="toolTip" >
               <string>Click to select a different color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_21" >
           <property name="text" >
            <string>Cursor Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ColorButton" name="mp_cursorColorBtn" >
           <property name="toolTip" >
            <string>Click to select a different color</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okBtn" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_okBtn</sender>
   <signal>clicked()</signal>
   <receiver>UserPreferences</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>479</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel" >
     <x>263</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
